Microsoft.Toolkit.Uwp.UI.Animations/Extensions/AnimationExtensions.cs

```diff
@@ -133,5 +132,34 @@ private static CompositeTransform GetAttachedCompositeTransform(UIElement elemen
 
             return result;
         }
+
+        private static EasingFunctionBase GetEasingFunction(InterpolationType interpolationType)
```

Comment:
This only works for storyboards. Can you also add support for composition animations, like blur? You will most likely need to have two functions, one for composition and one for storyboards